internal/rego/matchers.go

```diff
@@ -25,13 +28,23 @@ type KindMatcher struct {
 	Kind     string
 }
 
+// MatchLabelsMatcher is the matcher to generate `constraints.spec.match.labelSelector.matchLabels`.
+type MatchLabelsMatcher map[string]string
+
 // Matchers returns all of the matchers found in the rego file.
 func (r Rego) Matchers() Matchers {
 	var matchers Matchers
 	for _, comment := range r.comments {
 		if strings.HasPrefix(comment, "@kinds") {
 			matchers.KindMatchers = getKindMatchers(comment)
 		}
+		if strings.HasPrefix(comment, "@matchlabels") {
+			var err error
+			matchers.MatchLabelsMatcher, err = getMatchLabelsMatcher(comment)
+			if err != nil {
+				log.Fatal(err)
```

Comment:
Opinion: Functions in packages should not end execution, they should only return errors back to the caller. This function should be updated to also return an `error`.

Reply:
I fixed it!